src/components/ProfileImage.tsx

```diff
@@ -0,0 +1,162 @@
+import i18n from 'lib/i18n';
+import React from 'react';
+import {
+  Image,
+  StyleProp,
+  StyleSheet,
+  Text,
+  TextStyle,
+  View,
+  ViewStyle,
+} from 'react-native';
+const defaultSize = 80;
+const defaultFontSize = 32;
+
+export interface Props {
+  style?: StyleProp<ViewStyle>;
+  textStyle?: StyleProp<TextStyle>;
+  size?: number;
+  user: {
+    userId: string;
+    profile?: {
+      displayName: string | null;
+      picture?: string | null;
+    };
+  };
+  customBackgroundStyle?: StyleProp<ViewStyle>;
+  imageUri?: string;
+}
+
+export const imageWrapperStyles: StyleProp<ViewStyle> = {
+  backgroundColor: '#02BFF1', // light blue
```

Comment:
Perhaps we can default to this color when useBranding is introduced and the specified color is not defined in the branding, i.e, `brand.styles.image.borderColor ?? '#02BFF1',`